### PR TITLE
Fix popover and breadcrumbs menu

### LINKED ISSUE #124: Fix content area popout windows and breadcrumbs menu
When a content area popout overlaps an unpopulated content area, bottom options are not clicable ([see video](https://user-images.githubusercontent.com/48313701/153563409-2977d29b-86b4-4b51-b88d-39af8e5b71e7.mp4))

`breadcrumbs-menu-sd` component needs some improvements:

- Show correct page as current page.
- When the store mount point is not front page (root), then do not show site root it in breadcrumbs menu.

When product uses variants and variant prices are different, then the price should be shown on the buy button when a variant is selected.

Auto-gen product page title alignment should be editable through design editor.

### PR DESCRIPTION
**Pull request by PeeterPaal**
Closes #124

### CODE REVIEW

File: components/menu-breadcrumbs-sd.tpl [hunk 1]
```diff
@@ -6,7 +6,7 @@
     {%- assign pageUrl = page.url | remove_first: "/" -%}
```
Comment (pxska) [inline components/menu-breadcrumbs-sd.tpl:6]: `pageUrl` isn't necessary in this context anymore.

**Review by pxska — APPROVED** [review]
One small comment, otherwise looks good!

File: components/menu-breadcrumbs-sd.tpl [hunk 2]
```diff
@@ -1,12 +1,16 @@
...
+  {%- assign breadcrumbsOffset = 1 -%}
+{%- else -%}
+  {%- assign breadcrumbsOffset = 0 -%}
+{%- endif -%}
 
 <ul class="menu menu-horizontal menu-public menu-breadcrumbs" data-search-indexing-allowed="false">
-  {%- for listItem in breadcrumbsObj.itemListElement %}
-    {%- assign pageUrl = page.url | remove_first: "/" -%}
-
+  {%- assign rootShown = true -%}
```
Comment (tanelj) [inline components/menu-breadcrumbs-sd.tpl:10]: Unused variable
```suggestion
  {%- assign rootShown = true -%}
```

**Review by tanelj — CHANGES_REQUESTED** [review]
Nice, but one unused variable.

**Review by tanelj — APPROVED** [review]

**Review by pxska — APPROVED** [review]

**Pull request merged**